normal.py

```diff
@@ -59,6 +59,15 @@ def __call__(self, /, *args, **kwargs):
             result = function(result)
         return result
 
+    def __add__(self, other):
+        """Compose using '+' operator."""
+        if isinstance(other, compose):
+            return compose(*self.functions, *other.functions)
+        elif callable(other):
+            return compose(*self.functions, other)
```

Comment:
Now that `acompose` is fixed to not be a subclass of `compose`, everything in this comment still applies, it's just that now `acompose` would have to explicitly reuse the same implementation instead of implicitly inheriting it, like it now does with `__init__`, `__repr__`, and `functions`.